### PR TITLE
change models for visualizations

### PR DESCRIPTION
**Pull request by tcnichol**
This pull request changes models for visualizations based on feedback.

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]

File: backend/app/models/visualization_config.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ class VisualizationConfigBase(BaseModel):
     extractor_info: Optional[ExtractorInfo]
     job: Optional[EventListenerJobDB]
     client: Optional[str]
-    viz_config_data: dict = {}
-    # TODO add json document or key value pairs, config_variables
+    parameters: dict = {}
+    visualization_bytes_id: List[PyObjectId]
```
Comment (longshuicy) [inline backend/app/models/visualization_config.py:20]: maybe plural  `_ids` since it's a list

File: backend/app/models/visualization.py [hunk 2]
```diff
@@ -24,21 +24,21 @@ def validate_to_json(cls, value):
...
+class VisualizationDataDB(Document, VisualizationDataBase):
     creator: UserOut
     created: datetime = Field(default_factory=datetime.utcnow)
     modified: datetime = Field(default_factory=datetime.utcnow)
     bytes: int = 0
     content_type: ContentType = ContentType()
 
     class Settings:
-        name = "visualizations"
+        name = "visualizationData"
```
Comment (lmarini) [inline backend/app/models/visualization.py:39]: Do you we use camel case anywhere? Should we use all lower case with underscore between each word instead?

**Review by lmarini — CHANGES_REQUESTED** [review]

**Comment by ddey2** [comment]
@tcnichol 
Should we change "vizconfig" in 29 in isualization_config.py to "visualization_config" may be?

File: backend/app/models/visualization.py (hunk 2 above)
Comment (tcnichol) [reply backend/app/models/visualization.py:39]: I think that will be better. Changed.

**Comment by longshuicy** [comment]
Could you resolve merge conflict?

**Review by longshuicy — APPROVED** [review]

**Review by max-zilla — APPROVED** [review]

File: backend/app/routers/visualization.py
Comment (tcnichol) [inline backend/app/routers/visualization.py:1]: The credentials are removed. They were causing this route to give a 403 error.

**Pull request merged**